.github/workflows/benchmark.yml

```diff
@@ -10,7 +10,7 @@ jobs:
       fail-fast: false
       matrix:
         node_version:
-          - "18"
+          - "20"
```

Comment:
Did you want to put this back to node 18 if testing with node v20 is pending?

Reply:
only the http instrumentation test is failing for 20. The rest of the SDK is fine so I think its ok to keep this here.

Reply:
actually yes i'll bump it back

Reply:
has something changed that this is not back at node 18? 🤔

Reply:
resolving as it's back at 18 in 543f0b470c2b7dc87b38cd4c3948aa3cfb61c41b